packages/forms-web-app/src/controllers/lpa-dashboard/email-address.js

```diff
@@ -10,13 +10,19 @@ const {
 const getEmailAddress = async (req, res) => {
 	const user = getLPAUserFromSession(req);
 	const lpa = await getLPA(user.lpaCode);
+	req.session.addUserLpaDomain = lpa.domain;
```

Comment:
Now I've seen this I'm thinking we may need more details about the lpa throughout the process, so potentially may be sensible to do this lpa lookup on login and store on the user itself, so we can then do something like user.lpa.domain, and not need additional api calls

Don't have to do on this PR though

Reply:
agreed - perhaps a new ticket?